microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md

```diff
@@ -28,19 +28,29 @@ In Windows 11, Progressive Web Apps (PWAs) can define widgets, update them, and
 
 Widgets are experimentally supported starting with Microsoft Edge 108.
 
-To enable widget support in Microsoft Edge for your website:
+#### Enable local development of widgets
 
-1. Enroll your site in the [Web App Widgets](https://developer.microsoft.com/microsoft-edge/origin-trials/web-app-widgets/registration/) origin trial. To learn more about origin trials, see [how to enroll your site in an origin trial](origin-trials.md#enroll-your-site-in-an-origin-trial).
+To enable local development of widgets:
```

Comment:
```suggestion
To enable developing widgets on your local machine:
```
hints to define "local development" instead of duplicating the new term twice